src/controller/python/chip/native/__init__.py

```diff
@@ -18,6 +20,39 @@ def _AllDirsToRoot(dir):
         dir = parent
 
 
+class PyChipError(ctypes.Structure):
```

Comment:
It would be good if we could provide getters that could extract out the various sub-parts of the error so that Python logic that wants to look for specific errors could more easily do so.

Reply:
Getters are implemented, mark as resolved.